tools/md2yml.py

```diff
@@ -0,0 +1,214 @@
+import argparse
+import os
+import os.path as osp
+import re
+
+import yaml
```

Comment:
Better to use mmcv.dump to dump yaml files. Please double-check the generated metafiles are alright.